test/test_ff_utils.py

```diff
@@ -4,6 +4,8 @@
 from dcicutils import ff_utils
 pytestmark = pytest.mark.working
 
+MASTERTEST_URL = 'http://fourfront-mastertest.9wzadzju3p.us-east-1.elasticbeanstalk.com/'
```

Comment:
I've updated the test so that it does something a little strange IMO to properly resolve this url using integrated_ff. Might want to look to see if you think its okay.